README.md

```diff
@@ -1,18 +1,101 @@
-SDK for interacting with the Phoenix program.
+#Phoenix SDK
 
-We currently have a Rust and Typescript SDK, with Python on the way.
+An SDK for interacting with the Phoenix program.
+
+**We currently support Rust and TypeScript, with Python on the way.*
+
+## Rust
 
 To run the sample Rust code, run:
 
-```
+```sh
 $ cd rust
 $ cargo run --bin sample -- -r $YOUR_DEVNET_RPC_ENDPOINT
 ```
 
-To run the sample typescript code, run:
+## TypeScript
 
+```TypeScript
+import { ASSOCIATED_TOKEN_PROGRAM_ID, TOKEN_PROGRAM_ID } from "@solana/spl-token";
+import { Connection, Keypair, Transaction } from "@solana/web3.js";
+import * as Phoenix from "@ellipsis-labs/phoenix-sdk";
+
+async function exampleSwap() {
+  const connection = new Connection("https://api.devnet.solana.com/");
+  const trader = new Keypair();
+
+  // Creating a Phoenix client
+  const phoenix = await Phoenix.Client.create(connection);
+  const market = phoenix.markets.find((market) => market.name === "SOL/USDC");
+
+  // Example of a simple swap
```

Comment:
add note: this swaps 1 USDC for SOL (or perhaps it's the other way around) at the current price with 0.5% slippage tolerance